scipy/linalg/tests/test_matfuncs.py

```diff
@@ -15,7 +15,8 @@
 
 import scipy.linalg
 from scipy.linalg import (funm, signm, logm, sqrtm, fractional_matrix_power,
-                          expm, expm_frechet, expm_cond, norm, khatri_rao)
+                          expm, expm_frechet, expm_cond, norm, khatri_rao,
+                          cosm, sinm, tanm, coshm, sinhm, tanhm)
```

Comment:
I guess it's the pattern in `linalg` to import all the functions to be tested in the file. These functions aren't currently tested anywhere, actually, so it seems we need to add the imports somewhere, and this seems like the appropriate file.